### DISCUSSION TITLE
notes/eddie-zaneski/

### DISCUSSION THREAD

**Discussion by giscus** [dendronhq/cascadia-js-2022#10] [Announcements]
# notes/eddie-zaneski/

Notes about the cascadiajs 2022 conference

http://localhost:3000/notes/dwane-hemmings/

**Comment by vim-usds** [comment]
Slides here - https://docs.google.com/presentation/d/18iaI3qXNZ5CBeZ2Z7Xtt4wgAZGeya0q12DrpRqyYilo/edit#slide=id.g1248a1b3c72_0_35
